macros/schema_tests/cardinality_equality.sql

```diff
@@ -0,0 +1,28 @@
+{% macro test_cardinality_equality(model, from, to, field) %}
+
+with table_a as (
+select
+  count(1) as num_rows,
+  {{ from }}
+from {{ model }}
+group by 2
+),
+
+table_b as (
+select
+  count(1) as num_rows,
+  {{ field }}
+from {{ to }}
+group by 2
+)
+
+select count(1)
+from (
+select *
+from table_a
+except
```

Comment:
This is groovy! I think the `except` set function is asymmetrical -- `table_a - table_b` is different than `table_b - table_a`. As a result, this test would "pass" if `table_b` contained records which were not present in `table_a`, ie. if `table_b` is a superset of `table_a`.

There may be a simpler way to do this, but I've done something like this before:

```sql
-- table_a and table_b are equal if this union results in 0 rows
(
  select * from table_a
  except
  select * from table_b
)

union all

(
  select * from table_b
  except
  select * from table_a
)
```

This is cool (for some value of cool) b/c it's essentially the definition of set equality:
A = B if A is a subset of B, and B is a subset of A

Reply:
Good catch! It does feel a little bit strange that there isn't a simpler way to do this than a `union all`